hazelcast/src/main/java/com/hazelcast/internal/util/OperatingSystemMXBeanSupport.java

```diff
@@ -43,6 +55,10 @@ private OperatingSystemMXBeanSupport() {
     public static long readLongAttribute(String attributeName, long defaultValue) {
         try {
             String methodName = "get" + attributeName;
+            if (methodName.equals("getFreePhysicalMemorySize") && GET_FREE_PHYSICAL_MEMORY_SIZE_DISABLED) {
```

Comment:
Minor: consider changing to `if (GET_FREE_PHYSICAL_MEMORY_SIZE_DISABLED && methodName.equals("getFreePhysicalMemorySize")` to avoid unnecessary `equals()` calls here as well (the order is fine in the other places).

Reply:
Done